src/azure-cli/azure/cli/command_modules/storage/_help.py

```diff
@@ -40,13 +40,19 @@
 long-summary: >
     Update the properties of a storage account's blob service, including
     properties for Storage Analytics and CORS (Cross-Origin Resource
-    Sharing) rules. But currently we only support enabling or disabling change feed for a storage account's blob service.
+    Sharing) rules. But currently we only support enabling or disabling 'change feed' and 'delete retention policy' for a storage account's blob service.
```

Comment:
I think we can support all the properties with generic update.

Reply:
Ok, I've removed this code comment